examples/multisig_plain/lib.rs

```diff
@@ -0,0 +1,231 @@
+#![cfg_attr(not(feature = "std"), no_std)]
+
+use ink_lang as ink;
+
+#[ink::contract(version = "0.1.0", env = MyEnv)]
```

Comment:
It did not work outside of the module. I guess the user of this type will be generated inside of he module. Maybe with an absolute path it would work when the type is declared outside.

Reply:
I am not sure how to progress with this item. Should I crate an issue for this? I do not find that behavior surprising. (I am trying to get all items to resolved).

Reply:
outdated comments from what I can tell. resolved